tests/bud.bats

```diff
@@ -4425,9 +4425,66 @@ EOM
 }
 
 @test "bud-implicit-no-history" {
-  _prefetch nixery.dev/shell
-  run_buildah build $WITH_POLICY_JSON --layers=false $BUDFILES/no-history
-  run_buildah build $WITH_POLICY_JSON --layers=true  $BUDFILES/no-history
+  _prefetch busybox
```

Comment:
On a repush, would you consider replacing all of this with
```
  testimage=quay.io/libpod/buildah-testimage-nohistory:20240422
  _prefetch $testimage
...
  check that
  run buildah on it
```

Reply:
That'll _probably_ work.  I say probably because I only just realized that hard-coding the architecture of the image as "amd64" could create problems if any of the binaries being run using RUN instructions aren't coming from the current architecture's "busybox" image.

Reply:
Gotcha - I hadn't noticed/remembered that there were `RUN`s in the containerfile. I need to rethink this. Tomorrow.